extensions/wikia/AdEngine/js/template/bfaaMobile.js

```diff
@@ -12,6 +13,7 @@ define('ext.wikia.adEngine.template.bfaaMobile', [
 ], function (
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Function has 10 parameters which is greater than 7 authorized. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.wikia-inc.com/coding_rules#rule_key=javascript%3AExcessiveParameterList)